charts/spire/charts/spiffe-oidc-discovery-provider/templates/_helpers.tpl

```diff
@@ -85,4 +85,3 @@ Create the name of the service account to use
 {{- define "spiffe-oidc-discovery-provider.workload-api-socket-path" -}}
 {{- printf "/spiffe-workload-api/%s" .Values.agentSocketName }}
 {{- end }}
```

Comment:
This additional dash is just a bug, and not part of this effort.  Next time submit a fix for it in a different PR.

Reply:
Its not a dash. it looks like an editor removed the blank line at the end of the file. I can undo that.